### DISCUSSION TITLE
How can I chage the directory to where the computers ar located?

### DISCUSSION THREAD

**Discussion by zaze06** [MCJack123/craftos2#202] [Q&A]
As stated in the title `How can I chage the directory to where the computers ar located?`
I what to edit file is vs code and have a hard time finding the location to where the computer files ar located (I may use a shortcut to the map but I docent do this to often) and it whole be esier if I could just point the to the vs code workspace and it generates the files and folders there

**Comment by MCJack123** [comment]
You can use the `-d` command-line option to set the save directory for a session. Alternatively, you can use `--start-dir` to set the path for the first computer opened. (See https://www.craftos-pc.cc/docs/cli for more flags you can use.)

Also, there's a built-in command to open a computer in the extension. Press Shift+Ctrl+P and type in `CraftOS-PC: Open Data Directory for Computer...`, then type the ID of the computer to open (usually 0). Then the data folder for the computer will be opened.
